pyjl/plugins.py

```diff
@@ -129,32 +117,24 @@ def visit_asyncio_run(node, vargs) -> str:
 
 # small one liners are inlined here as lambdas
 SMALL_DISPATCH_MAP = {
-    "str": lambda n, vargs: f"&{vargs[0]}.to_string()",
-    "len": lambda n, vargs: f"{vargs[0]}.len()",
+    "str": lambda n, vargs: f"string({vargs[0]})",
+    "len": lambda n, vargs: f"length({vargs[0]})",
     "enumerate": lambda n, vargs: f"{vargs[0]}.iter().enumerate()",
     "sum": lambda n, vargs: f"{vargs[0]}.iter().sum()",
-    "int": functools.partial(JuiliaTranspilerPlugins.visit_cast, cast_to="i32"),
-    "bool": lambda n, vargs: f"({vargs[0]} != 0)",
-    "float": functools.partial(JuiliaTranspilerPlugins.visit_cast, cast_to="f64"),
-    # as usize below is a hack to pass comb_sort.rs. Need a better solution
-    "floor": lambda n, vargs: f"{vargs[0]}.floor() as usize",
-    "reversed": lambda n, vargs: f"{vargs[0]}.iter().rev()",
-    "map": lambda n, vargs: f"{vargs[1]}.iter().map({vargs[0]})",
-    "filter": lambda n, vargs: f"{vargs[1]}.into_iter().filter({vargs[0]})",
-    "list": lambda n, vargs: f"{vargs[0]}.collect::<Vec<_>>()",
-    "asyncio.run": JuiliaTranspilerPlugins.visit_asyncio_run,
+    "bool": lambda n, vargs: f"Bool({vargs[0]})",
+    # ::Int64 below is a hack to pass comb_sort.jl. Need a better solution
+    "floor": lambda n, vargs: f"Int64(floor({vargs[0]}))",
 }
 
 SMALL_USINGS_MAP = {
     "asyncio.run": "futures::executor::block_on",
 }
 
 DISPATCH_MAP = {
-    "max": functools.partial(JuiliaTranspilerPlugins.visit_min_max, is_max=True),
```

Comment:
Add these back, so the tests you added pass for Julia?

Reply:
The tests do pass; these were wrong and unnecessary copy and paste.